examples/named_entity_recognition/ner_wikigold_transformer.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>Would be nice to show the evaluation results. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Added scrapbook.</p>